_posts/2018-04-16-seven-reasons-why-we-would-like-you-to-give-a-talk-at-euruko.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "7 reasons why we’d like you to give a talk at EuRuKo 2018"
+layout: post
+date: 2018-04-16 10:00:00 +0100
+image: vienna.jpg
+---
+
+Speaking at a conference for a large audience might sound a bit daunting at first, but at EuRuKo we like to welcome speakers from all backgrounds, experience and knowledge levels to give a talk. This could be about how you use Ruby, what you’ve learned while working with Ruby or how you see the Ruby community, or the developer community as a whole.
+
+With this list of 7 reasons we hope to make speaking at EuRuKo a positively unforgettable experience for you. And don’t forget to apply, **the deadline is April 17th!**
```

Comment:
We should add the time and maybe it's better to "And don't forget to apply asap, the deadline is April 17th 14:00 CEST. **Yes, that's tomorrow at lunch!**

Reply:
Yes please do!